hops/featurestore.py

```diff
@@ -455,29 +444,21 @@ def insert_into_featuregroup(df, featuregroup, featurestore=None, featuregroup_v
         core._do_insert_into_featuregroup(df, featuregroup,
                                           core._get_featurestore_metadata(featurestore, update_cache=False),
                                           featurestore=featurestore, featuregroup_version=featuregroup_version,
-                                          mode=mode, descriptive_statistics=descriptive_statistics,
-                                          feature_correlation=feature_correlation,
-                                          feature_histograms=feature_histograms, cluster_analysis=cluster_analysis,
-                                          stat_columns=stat_columns, num_bins=num_bins, corr_method=corr_method,
-                                          num_clusters=num_clusters, online=online, offline=offline)
+                                          mode=mode, online=online, offline=offline)
     except:
         # Retry with updated cache
         core._do_insert_into_featuregroup(df, featuregroup,
                                           core._get_featurestore_metadata(featurestore, update_cache=True),
                                           featurestore=featurestore, featuregroup_version=featuregroup_version,
-                                          mode=mode, descriptive_statistics=descriptive_statistics,
-                                          feature_correlation=feature_correlation,
-                                          feature_histograms=feature_histograms, cluster_analysis=cluster_analysis,
-                                          stat_columns=stat_columns, num_bins=num_bins, corr_method=corr_method,
-                                          num_clusters=num_clusters, online=online, offline=offline)
+                                          mode=mode, online=online, offline=offline)
 
 
-def update_featuregroup_stats(featuregroup, featuregroup_version=1, featurestore=None, descriptive_statistics=True,
-                              feature_correlation=True, feature_histograms=True, cluster_analysis=True,
-                              stat_columns=None, num_bins=20, num_clusters=5, corr_method='pearson'):
+def update_featuregroup_stats(featuregroup, featuregroup_version=1, featurestore=None, descriptive_statistics=None,
```

Comment:
In the new version of the API we decided to pass an object describing the statistics configuration. I think we can already start the transition with this PR.

Reply:
Ok, maybe not. Too much code to change

Reply:
Yes I was going to say the same. Also might be confusing for users of the current API